### PR TITLE
Selection Sort in JavaScript

### PR DESCRIPTION
**Pull request by ireneelizabethsabu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
Adding algorithm for selection sort in javascript
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#2505 
(Write your answer here.)

### CODE REVIEW

**Comment by ireneelizabethsabu** [comment]
@HarshCasper @Apoorve73 please review this PR.

File: JavaScript/sort/selectionSort.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        arr[minPos] = arr[i];
+        arr[i] = temp;
+    }
+    return arr;
+}
+//For  dynamic input use the below line.
+//var arr = prompt("Enter numbers to sort comma separated: ").split(",");
+var arr = [9, 8, 7, 6, 5, 4, 3];
+console.log(selectionSort(arr));
+//Expected output : [3,4,5,6,7,8,9]
```
Comment (HarshCasper) [inline JavaScript/sort/selectionSort.js]: Add a new line at the EOF

File: JavaScript/sort/selectionSort.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        }
+        //swapping current element and smallest element;
+        let temp = arr[minPos];
+        arr[minPos] = arr[i];
+        arr[i] = temp;
+    }
+    return arr;
+}
+//For  dynamic input use the below line.
+//var arr = prompt("Enter numbers to sort comma separated: ").split(",");
```
Comment (HarshCasper) [inline JavaScript/sort/selectionSort.js]: Prefer Dynamic Input instead of static input

File: JavaScript/sort/selectionSort.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+/*
+SELECTION SORT:
+    Selection sort is a simple inplace sorting algorithm.
+    Time Complexity - O(n^2)
+    Space Complexity - O(1)
```
Comment (HarshCasper) [inline JavaScript/sort/selectionSort.js]: Add description of Selection Sort here. Shift Time/Space Complexity to the end

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by ireneelizabethsabu** [comment]
@HarshCasper Kindly look into the changes made.Thank you.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**